instrumentation/src/com/intellij/rt/coverage/instrumentation/Instrumentator.java

```diff
@@ -52,21 +52,30 @@ public void performPremain(String argsString, Instrumentation instrumentation) t
       return;
     }
 
-    final boolean traceLines = args.length > 0 && Boolean.valueOf(args[1]);
-    final boolean sampling = args.length == 0 || Boolean.valueOf(args[4]);
+    if (0 < args.length && args.length < 5) {
```

Comment:
no args is also tolerated. Not sure what it means though...

Reply:
In current code no args is a possible case. The coverage will run but no coverage report will be generated, so it seems useless. I decided that it is some special important case as many checks `args.length > 0 ? ... : ...` are implemented.

Can I take into account only the case when `args.length >= 4`?

Reply:
I don't remember why no-arg case was introduced. I don't suggest to change anything